Winapp2.ini

```diff
@@ -13122,13 +13122,11 @@ Detect=HKCU\Software\Classes\Local Settings\Software\Microsoft\Windows\CurrentVe
 FileKey1=%LocalAppData%\Packages\Microsoft.LockApp_*\AC\AppCache|*.*|RECURSE
 FileKey2=%LocalAppData%\Packages\Microsoft.LockApp_*\AC\INet*|*.*|RECURSE
 FileKey3=%LocalAppData%\Packages\Microsoft.LockApp_*\AC\Microsoft\CryptnetUrlCache\*|*.*|RECURSE
-FileKey4=%LocalAppData%\Packages\Microsoft.LockApp_*\AC\NVIDIA Corporation\NV_Cache|*.*
-FileKey5=%LocalAppData%\Packages\Microsoft.LockApp_*\AC\NVIDIA\DXCache|*.*

```

Comment:
the NVIDIA directories exist on my machine 
![image](https://user-images.githubusercontent.com/529203/145817459-d1ce8165-dc39-4f1e-8de6-6aac30e99ba1.png)

Reply:
These directories are "globally" cleaned by the entries [NVIDIA *] (and [AMD/ATI *]).